models/migrations/v45.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/go-xorm/xorm"
+)
+
+func removeIndexColumnFromRepoUnitTable(x *xorm.Engine) (err error) {
+	if _, err := x.Exec("ALTER TABLE repo_unit DROP COLUMN index"); err != nil {
```

Comment:
Columns can not be dropped on sqlite. Please add if to switch like in this migration: https://github.com/go-gitea/gitea/blob/1739e84ac02c0384c04576a00abab9348293f9c7/models/migrations/v40.go#L43

Reply:
Done